assets/scss/data-land.scss

```diff
@@ -11,6 +11,6 @@
   }
 }
 
-.supplemental-info-description {
+.pre-line-white-space {
```

Comment:
Don't we usually merge develop into the epic branch rather than a feature branch that is going into the epic